Services/WebAccessChecker/classes/class.ilWACToken.php

```diff
@@ -89,7 +89,7 @@ public function generateToken() {
 		$this->initSalt();
 		$token = implode('-', array(
 			self::getSALT(),
-			$this->getIp(),
+			//$this->getIp(),
```

Comment:
Could you please remove the line completely instead of just commenting it out? thanks in advance!

Reply:
Done